config/settings/development.yml

```diff
@@ -9,8 +9,8 @@ betamocks:
   enabled: true
   recording: false
   # the cache dir depends on how you run the api, run `bin/spring stop` after switching this setting
-  # cache_dir: ../vets-api-mockdata # via rails; e.g. bundle exec rails s or bundle exec rails c
-  cache_dir: /cache # via docker; e.g. make up or make console
+  cache_dir: ../vets-api-mockdata # via rails; e.g. bundle exec rails s or bundle exec rails c
+  # cache_dir: /cache # via docker; e.g. make up or make console
```

Comment:
I don't believe we want to change this value. Was this committed by accident?

Reply:
Yeah good catch that was an accidentally committed when fixing rubocop issues and then committed all changes on the local branch. Reverting this file from master and pushing back up.